internal/proto/replication.proto

```diff
@@ -174,3 +177,31 @@ message ResyncVolumeResponse{
   // This field is REQUIRED.
   bool ready = 1;
 }
+
+// getVolumeReplicationInfoRequest holds the required information to get the Volume replication info.
+message GetVolumeReplicationInfoRequest {
+  // The identifier for this volume, generated by the plugin during
+  // CreateVolume CSI RPC call.
+  // This field is REQUIRED.
+  // This field MUST contain enough information to uniquely identify
+  // this specific volume vs all other volumes supported by this plugin.
+  // This field SHALL be used by the CO in subsequent calls to refer to
+  // this volume.
+  string volume_id = 1;
+  // The identifier for the replication.
+  // This field is OPTIONAL.
+  // This field MUST contain enough information, together with volume_id,
+  // to uniquely identify this specific replication
+  // vs all other replications supported by this plugin.
+  string replication_id = 2;
+  // Secrets required by the plugin to complete the request.
+  string secret_name = 3;
+  string secret_namespace = 4;
+}
+// GetVolumeReplicationInfoResponse holds the information to send the
+// volume replication info.
+message GetVolumeReplicationInfoResponse{
+  // Holds the last sync time.
+  // This field is REQUIRED.
+  int64 last_sync_time = 1;
```

Comment:
https://github.com/csi-addons/spec/pull/44/files#diff-36da0b8129c7cf1c1695bf0ae42ea29185aa75ee81a8a34b7ad1a5d1eedd989f

Use timestamp here too.

Reply:
This is still not implemented, 
with this change, we can remove some code for conversion in other functions too.